### PR TITLE
New data engineering test

### PR DESCRIPTION
**Pull request by mickey**

### CODE REVIEW

**Review by faouzelfassi — APPROVED** [review]

**Review by faouzelfassi — APPROVED** [review]

File: data/level_file [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+for _ in itertools.repeat(None, 1000):
+  id = uuid.uuid4()
+
+  filename = f'./logs/{id}.txt'
+  os.makedirs(os.path.dirname(filename), exist_ok=True)
+
+  with open(filename, 'w') as file:
+    file.write(log_generator.sample(id))
+
+print("Generated 1000 files in ./logs")
```
Comment (godric) [inline data/level_file:18]: 👍

File: data/lib/http_emitter.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import itertools
+import requests
+import uuid
+
+import lib.log_generator as log_generator
+
+ENDPOINT = "http://127.0.0.1:3000"
+
+for _ in itertools.repeat(None, 1000):
```
Comment (godric) [inline data/lib/http_emitter.py]: 💎 Ruby!

I think the more standard way would be
```
for i in range(100):
``` 
Pretty sure a python linter would not complain about having a un-used variable in this context

File: data_legacy/README.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# Drivy Data Challenge
```
Comment (godric) [inline data_legacy/README.md]: Not convinced that keeping safety copies of old files in a git repository makes a lot of sense.

**Review by godric — COMMENTED** [review]
🤷‍♂️

**Pull request merged**